scipy/_lib/tests/test__util.py

```diff
@@ -388,6 +389,26 @@ def mock_str():
     assert res == ref
 
 
+@_transition_to_rng("seed", position_num=1, replace_doc=False)
```

Comment:
In case you're interested, there were some more comprehensive tests merged with the SPEC, originally:
https://github.com/stefanv/specs/blob/118019035e6a5e591917dfa0ea829027d40f839a/spec-0007/test_transition_to_rng.py

That said, I don't have a strong opinion about whether we need thorough tests of every aspect of the code in SciPy since we're basically just vendoring it.